### ISSUE TITLE
Cookie "expires" is never set

### ISSUE THREAD

**Issue by awhitty** [lynndylanhurley/ng-token-auth#119]
I may be interpreting this the wrong way, but in order for the user's logged-in session to last longer than the browser session, the cookie storing the auth tokens should have an `expires` property. #107 mentions this issue, but the issue was closed, and I do not think it has been adequately addressed.

As soon as a user refreshes the page, the user's session and relevant cookies have been lost, forcing the user to reauthenticate.

If it helps, I'm using the email/password authentication scheme. 

The relevant code:
```js
persistData: function(key, val, configName) {
  switch (this.getConfig(configName).storage) {
    case 'localStorage':
      return $window.localStorage.setItem(key, JSON.stringify(val));
    default:
      return ipCookie(key, val, {
        path: '/'
      });
  }
},
```

You can see that there is no `expires:` in the object passed to ipCookie. Is this the correct behavior?

**Comment by lynndylanhurley** [comment]
@awhitty - the token expiry is enforced by the server, so the cookies should remain in the browser indefinitely.

I did just find this in the ipCookie docs:

>  If you don't specify the expiry date the cookie is trashed when you close the browser.

I don't remember it working this way initially - I may have built against a version that didn't have this behavior.

Thanks for reporting the issue, I'll push a fix ASAP.

**Comment by proletarian** [comment]
Just checking if this is still imminent. Thanks!

**Comment by lynndylanhurley** [comment]
Hi @proletarian - I'll push a fix for this tonight. Sorry for the delay.

**Comment by proletarian** [comment]
Woohoo! No worries. Appreciate the update.

On Monday, 18 May 2015, Lynn Dylan Hurley <notifications@github.com> wrote:

> Hi @proletarian <https://github.com/proletarian> - I'll push a fix for
> this tonight. Sorry for the delay.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lynndylanhurley/ng-token-auth/issues/119#issuecomment-103175470>
> .
>


-- 
Best regards,

*Adeel Ahmad*
CEO / Shopseen <https://www.shopseen.com>
Shopseen simplifies commerce.

*Follow @shopseen everywhere*
Instagram <http://instagram.com/shopseen> / Pinterest
<http://pinterest.com/shopseen> / Twitter <http://twitter.com/shopseen> /
Facebook <http://facebook.com/shopseen>

**Comment by lynndylanhurley** [comment]
Fixed in `v0.0.25`.
